web/pdf_viewer.js

```diff
@@ -2098,15 +2123,19 @@ class PDFViewer {
     this.#annotationEditorUIManager.updateParams(type, value);
   }
 
-  refresh() {
+  refresh(update, updateArgs = Object.create(null)) {
```

Comment:
This is actually a breaking change, so please adjust this as below and update the call-sites accordingly.
```suggestion
  refresh(noUpdate = false, updateArgs = Object.create(null)) {
```

Reply:
This still requires follow-up adjustments to all call-sites, otherwise I don't thing that things will work :-)